test/test_datasets.py

```diff
@@ -1790,5 +1790,78 @@ def test_targets(self):
                 assert item[6] == i // 3
 
 
+class LFWPeopleTestCase(datasets_utils.DatasetTestCase):
+    DATASET_CLASS = datasets.LFWPeople
+    FEATURE_TYPES = (PIL.Image.Image, int)
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(
+        train=(True, False),
```

Comment:
Currently the CI is failing because with this the test is trying to create `LFWPeople(..., train=True, ...)` which is no longer possible. This should be 
```suggestion
        split=("train", "test", "10fold"),
```
right?